doc/classes/ProjectSettings.xml

```diff
@@ -2383,9 +2383,10 @@
 			[b]Note:[/b] This property is only read when the project starts. To change the maximum number of simulated physics steps per frame at runtime, set [member Engine.max_physics_steps_per_frame] instead.
 		</member>
 		<member name="physics/common/physics_interpolation" type="bool" setter="" getter="" default="false">
-			If [code]true[/code], the renderer will interpolate the transforms of physics objects between the last two transforms, so that smooth motion is seen even when physics ticks do not coincide with rendered frames. See also [member Node.physics_interpolation_mode] and [method Node.reset_physics_interpolation].
-			[b]Note:[/b] If [code]true[/code], the physics jitter fix should be disabled by setting [member physics/common/physics_jitter_fix] to [code]0.0[/code].
+			If [code]true[/code], the renderer will interpolate the transforms of objects (both physics and non-physics) between the last two transforms, so that smooth motion is seen even when physics ticks do not coincide with rendered frames. See also [method Node.reset_physics_interpolation].
+			[b]Note:[/b] Although this is a global setting, individual branches of the [SceneTree] can be switched on and off using [member Node.physics_interpolation_mode], to give finer control.
```

Comment:
Although it's correct I personally avoid talking about branches in the SceneTree because it's rarely said in the class reference and it's a bit awkward to localize.
It's not perfect, but perhaps...?
```suggestion
			[b]Note:[/b] Individual nodes in the [SceneTree] can also toggle interpolation using [member Node.physics_interpolation_mode] for themselves and their children, for finer control.
```
Also this note should probably also exist in SceneTree's property for consistency.

Reply:
Branch is imo the best term, in graph theory and computer science it refers to a subset of the scene graph and its descendants, and it's easy to understand because of the analogy with a plant (that's why we call it a "tree").

In retrospect `control` would probably be better here than `switch` or `toggle`, as it's actually an enum : "OFF, ON, INHERIT".

I have changed in the `SceneTree`, but it's debatable whether this is really required as it refers to the `ProjectSetting`. Realistically it seems unlikely that anyone will be changing this who is not familiar with the project setting (and it may end up being removed, see #102762 ).